WordPress/src/androidTest/java/org/wordpress/android/e2e/flows/LoginFlow.java

```diff
@@ -38,6 +38,7 @@ public LoginFlow chooseContinueWithWpCom() {
     public LoginFlow enterEmailAddress(String emailAddress) {
         // Email Address Screen – Fill it in and click "Continue"
         // See LoginEmailFragment
+        clickOn(R.id.input);
```

Comment:
Great question! This is what I did initially, in a different PR. But there was a [valid point](https://github.com/wordpress-mobile/WordPress-Android/pull/16969#discussion_r934459065) from @ParaskP7 about potential unwanted effects of such approach. So, I added it only to places where I wanted to have it. Same here. I'd say, if there will be another PR where we need to add more `clickOn`s before entering text, it will make sense to make it a part of `populateTextField`.